src/utils.js

```diff
@@ -48,3 +48,20 @@ export function replaceEntry(newEntry) {
 export function identity(argument) { return argument }
 
 export function noop() {}
+
+/**
+ * Wraps an object and throws if an undefined property is accessed.
```

Comment:
Add explanation why this is used.
like `This way we ensure that no **undefined** case happens in the reducer`